openpype/hosts/nuke/startup/custom_popup.py

```diff
@@ -0,0 +1,174 @@
+import sys
```

Comment:
This is perhaps not the best name of the module. This seems to be something generic rather then specifically used for Read node start frame. How about to rename the module to something like `openpype_nuke_set_start_frame_to_reads.py`, this comunicate exacly the purpose of the script.